onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -523,26 +508,25 @@ static BilinearParams SetupUpsampleBilinear(int64_t input_height,
 }
 
 template <typename T>
-void UpsampleBilinear(int64_t batch_size,
-                      int64_t num_channels,
-                      int64_t input_height,
-                      int64_t input_width,
-                      int64_t output_height,
-                      int64_t output_width,
-                      float height_scale,
-                      float width_scale,
+void UpsampleBilinear(const int64_t batch_size,
```

Comment:
>UpsampleBilinear [](http://example.com/codeflow?start=5&length=16)

move this function to header file too for consistency?

Reply:
Moved